live-site/components/welcome-section/team-race/Team.styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+import { H3, P } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import { max } from '../../../../shared-ui/lib/responsive';
+
+const StyledTeamContainer = styled.div`
```

Comment:
If you add the other thing I mentioend for making the car on the line, then the rows are gonna look very close together - you can add some padding-top: 1em here to give it a bit more space.